src/config/dynamic-theme-fixes.config

```diff
@@ -11802,16 +11802,27 @@ body,
 nymag.com
 
 INVERT
-.logo-svg
+.article-feed-header svg
 .article-nav-top-center
 #latest-feed-title
+.logo-svg
 #site-feed-curbed-title > .feed-title-wrapper
 .tab-trigger > svg
 .tab-trigger > svg > circle
 .tab-trigger > svg > g > circle
+.title-logo
 #vulture-trigger > svg > g > path:nth-child(1)
 #wwwthecut-trigger > svg > .active > path
 
+CSS
+.collection-articles-lede_first-featured .lede-item[data-track-index="0"] .lede-image-wrap {
+    z-index: 0 !important;
+}
+.collection-articles-lede_first-featured .lede-item[data-track-index="0"] .lede-text-wrap {
+    z-index: 1 !important;
+    position: absolute !important;
```

Comment:
Just to be clear, this is a exception, mainly because due to the site structure + z-index's being in the play.